src/components/BuyerDashboard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Container, RecentOrders, ProductRequest, Notification, SmallBox } from '../components/mainComponent.jsx';
+import { Graph, People } from '../Images/MainImage.jsx';
+
+function BuyerDashboard() {
+    return (
+        <Container className="min-h-[45.5rem] bg-[#F1EAFF] flex flex-col pt-10 gap-10" px="px-0">
+            <Container className="flex flex-row justify-between">
+                <SmallBox
+                    imagesrc={Graph}
+                    title={"Total Orders"}
```

Comment:
Props should be passed for these, with appropriate default values

Reply:
required modifications are done.